app/controllers/RecruiterController.php

```diff
@@ -119,7 +119,8 @@ public static function home() {
         'company' => $companyprof,
         'stats' => $stats,
         'messages' => toJSON($messages),
-        'jobs' => $jobListings
+        'jobs' => $jobListings,
+        'id' => $myId
```

Comment:
We should follow the convention that if the id is a MongoId, we have it be called '_id' (with the underscore).